iPods were identified by a directory timestamp, but it changed each time
the ipod was mounted.  Instead identify a device by its FirewireGuid.
--- a/src/mediadevices/ipod/IpodDevice.cpp
+++ b/src/mediadevices/ipod/IpodDevice.cpp
@@ -77,8 +77,9 @@
 
     if ( m_uid.isEmpty() )
     {
-        QFileInfo f( m_mountPath + "/iPod_Control/Device" );
-        m_uid = f.created().toString( "yyMMdd_hhmmss" );
+        //QFileInfo f( m_mountPath + "/iPod_Control/Device" );
+        //m_uid = f.created().toString( "yyMMdd_hhmmss" );
+        m_uid = itdb_device_get_sysinfo( m_itdb->device, "FirewireGuid" );
 
         qDebug() << "uid" << m_uid;
     }
